src/components/layouts/EmptyState/DashboardEmptyState.tsx

```diff
@@ -54,6 +75,47 @@ const DashboardEmptyState = ({ userType }: { userType: UserType }) => {
           )}
         </Stack>
       </Box>
+      <Modal
+        open={showModal}
+        onClose={async () => {
+          store.dispatch(setShowModal())
+          store.dispatch(clearCreateTaskFields())
+          await bulkRemoveAttachments(attachments)
+        }}
+        aria-labelledby="create-task-modal"
+        aria-describedby="add-new-task"
+      >
+        <NewTaskForm
+          handleCreate={async () => {
+            if (title) {
+              store.dispatch(setShowModal())
+              store.dispatch(clearCreateTaskFields())
+              const createdTask = await handleCreate(
+                token as string,
+                CreateTaskRequestSchema.parse({
+                  title,
+                  body: description,
+                  workflowStateId,
+                  assigneeType,
+                  assigneeId,
+                  dueDate,
+                }),
+                true, //revalidation true for DashboadEmptyState only.
+              )
+              store.dispatch(appendTask(createdTask))
```

Comment:
I think it should be possible to use the response to update the tasks state in the store rather than revalidating somewhere and somewhere not revalidating.